src/hooks/use-posthog-analytics.ts

```diff
@@ -33,7 +37,9 @@ export default function usePostHogPageAnalytics(): void {
 
 	useEffect(() => {
 		// Ensures code only runs if PostHog has been initialized
-		if (!window?.posthog?.capture) return
+		if (!window?.posthog) return
+
+		window.posthog.config.capture_pageview = false
```

Comment:
This _should_ dedupe the events being sent to our posthog instance, we'll monitor after merging to see what the dataflow looks like